srcpkgs/tree-sitter/template

```diff
@@ -9,12 +9,12 @@ maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="https://tree-sitter.github.io"
 distfiles="https://github.com/tree-sitter/tree-sitter/archive/v${version}.tar.gz"
-checksum=b355e968ec2d0241bbd96748e00a9038f83968f85d822ecb9940cbe4c42e182e
+checksum=6181ede0b7470bfca37e293e7d5dc1d16469b9485d13f13a605baec4a8b1f791
 make_check=no # tests require generating fixtures from remote repositories
 
 post_patch() {
 	# fixes an indexmap error when cross compiling
-	cargo update --package autocfg:1.0.1 --precise 1.1.0
+	cargo update --package autocfg:1.1.0 --precise 1.1.0
```

Comment:
updating from 1.1.0 to 1.1.0?

Reply:
none of that is needed anymore, see https://github.com/void-linux/void-packages/pull/43292

Reply:
ty @icp1994, should I remove the tree-sitter commit from this PR altogether?